core/client/fs/src/main/java/alluxio/master/AlluxioMasterMonitor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master;
```

Comment:
`core/client/fs/src/main/java/alluxio/master/` is not the ideal location for this src file.
The reason is that the compiled class file of this src file  will go to the client jar we distribute. And we don't want to extend that jar unless it is necessary. In this case, as long as it stays in the assembly jar, we should be fine

You can put this file to `core/server/common/src/main/java/alluxio/master`. @aaudiber , what do you think?